### PR TITLE
docs: Add detailed description of how to run

### PR DESCRIPTION
**Pull request by andrei-bitca-dc**

### CODE REVIEW

**Review by izy-izzy — APPROVED** [review]

**Review by Mel0nHead — APPROVED** [review]

**Review by ayemonchit — APPROVED** [review]
Very nice 👍

**Review by ayemonchit — DISMISSED** [review]
@andrei-bitca-dc Can we also mention about set `startOnly` option to true if the build takes longer greater than 50 sec?

**Comment by andrei-bitca-dc** [comment]
> @andrei-bitca-dc Can we also mention about set `startOnly` option to true if the build takes longer greater than 50 sec?

Seems I forgot about those. Added!

File: azure-devops-extension/README.md [hunk 1]
```diff
@@ -8,6 +8,61 @@ It has the following tasks:
...
+- task: run-automator-build-task@1
+  inputs:
+    userKeyId: 'USER_KEY_ID' (string - generate new api key on deepcrawl core app)
+    userKeySecret: 'USER_KEY_SECRET' (string - generate new api key on deepcrawl core app)
+    testSuiteId: 'TEST_SUITE_ID' (string - use your created test suite id)
+    ciBuildId: 'CI_BUILD_ID' (optional - string)
+    startOnly: false (optional - boolean - use it if you want to skip polling)
+```
+
+*Note: Unless `startOnly` is set to `true`, max polling time is set to 50 minutes. If the build takes longer than 50 minutes, it will automatically fail the pipeline.*
```
Comment (ayemonchit) [inline azure-devops-extension/README.md:29]: Nice 👍

**Review by ayemonchit — APPROVED** [review]
Looks great

**Pull request merged**